site/content/docs/CICD/vercel.md

```diff
@@ -55,6 +55,47 @@ Once the project and the check/group have been linked, you are able to specify w
 
 Should you wish to unlink the Vercel project, simply click `Unlink this project`.
 
+## Targeting Preview deployments with `ENVIRONMENT_URL`
+
+By default, Vercel deploys your project as a **Preview deployment** using the unique **preview URL**. This URL is exposed
+to Checkly via a webhook and injected into the Checkly runtime as `ENVIRONMENT_URL`. 
+
+This means you can write a Browser check using Playwright that automatically targets the Preview deploy whenever this URL is 
+available, but otherwise just defaults to the stable **production URL**. 
+
+Here is a full example that we use ourselves to monitor the exact site your are on right now.
+
+
+```js {hl_lines=[7]}
+const { chromium } = require('playwright')
+
+(async () => {
+  const browser = await chromium.launch()
+  const page = await browser.newPage()
+
+  const targetUrl = process.env.ENVIRONMENT_URL || 'https://www.checklyhq.com'
```

Comment:
I'd rather the example URL be anything but checklyhq.com for more immediate understanding. Maybe example.com? https://demo.vercel.store?

Reply:
@ragog, maybe I should add our own website runs on Vercel? I kinda like showing exactly what we use ourselves i.e. dogfooding.

Reply:
@tnolet I see why that would be valuable. I am just trying to think about this from the perspective of a user who might be new to the integration, Vercel, Checkly, Playwright (possibly all the same time). Seeing the Checkly URL there might create confusion, e.g. "so I need Playwright to talk to Checkly, and this is how I do it." But the target needs to clearly be the user's own site. I try to be careful about these things when writing tuts, leaving the least amount of room for possible misunderstandings. A way to fight the Curse of Knowledge if you will.